schemas/com.olark/transcript/jsonschema/1-0-0

```diff
@@ -0,0 +1,209 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for an Olark transcript event",
+  "self": {
+    "vendor": "com.olark",
+    "name": "transcript",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "tags": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "maxLength": 128
+      }
+    },
+    "items": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "body": {
+            "type": "string"
+          },
+          "timestamp": {
+            "type": "string",
+            "maxLength": 64
+          },
+          "operatorId": {
+            "type": "string",
+            "maxLength": 64
+          },
+          "kind": {
+            "type": "string",
+            "maxLength": 64
+          },
+          "nickname": {
+            "type": "string",
+            "maxLength": 64
+          },
+          "visitor_nickname": {
+            "type": "string",
+            "maxLength": 64
+          }
+        },
+        "additionalProperties": false
+      }
+    },
+    "groups": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "kind": {
+            "type": "string",
+            "maxLength": 128
+          },
+          "name": {
+            "type": "string",
+            "maxLength": 128
+          },
+          "id": {
+            "type": "string",
+            "maxLength": 128
+          }
+        },
+        "additionalProperties": false
```

Comment:
Just in case - since we're inside the `array`, these `maxLength` and `additionalProperties` add almost no value - they cannot be mapped to Redshift types.

Reply:
But @chuwy they may be useful for some other storage target we support in the future?